tests/utilities/manage-demo-users.js

```diff
@@ -0,0 +1,123 @@
+
+config();
+
+export async function newDemoUser(newUserData) {
+  try {
+    const access_token = await getAccessToken();
+    const user = await createUser(access_token, newUserData);
+
+    //user returned to e2e test making the call
+    return user;
+  } catch (error) {
+    console.error(`Error: ${error}`);
+  }
+}
+
+export async function deleteDemoUser(userId) {
+  try {
+    const access_token = await getAccessToken();
+
+    await deleteUser(access_token, userId);
+  } catch (error) {
+    console.error(`Error: ${error}`);
+  }
+}
+
+async function getAccessToken() {
+  const AM_URL = process.env.VITE_FR_AM_URL;
+  const AdminUsername = process.env.VITE_ADMIN_USERNAME;
+  const AdminPassword = process.env.VITE_ADMIN_PASSWORD;
+
+  const myFrodo1 = frodo.createInstanceWithAdminAccount(AM_URL, AdminUsername, AdminPassword);
+  // destructure default instance for easier use of library functions
+  const { getTokens } = myFrodo1.login;
+  const { getInfo } = myFrodo1.info;
+
+  // login and obtain tokens
+  if (await getTokens()) {
+    // obtain and print information about the instance you are connected to
+    const info = await getInfo();
+    return info.bearerToken;
+  } else {
+    console.log('error getting tokens');
+  }
+}
+
+export async function getDemoUserId(userName) {
+  const AM_URL = process.env.VITE_FR_AM_URL;
+  const AdminUsername = process.env.VITE_ADMIN_USERNAME;
+  const AdminPassword = process.env.VITE_ADMIN_PASSWORD;
+
+  const myFrodo1 = frodo.createInstanceWithAdminAccount(AM_URL, AdminUsername, AdminPassword);
+
+  // destructure default instance for easier use of library functions
+  const { getTokens } = myFrodo1.login;
+
+  // login and obtain tokens
+  if (await getTokens()) {
+    const listOfUsers = await myFrodo1.idm.managed.queryManagedObjects('alpha_user', undefined, [
+      'userName',
+    ]);
+    const userId = listOfUsers.filter((res) => res.userName === userName).map((field) => field._id);
+    return userId;
+  } else {
+    console.log('error getting users');
+  }
+}
+
+async function createUser(access_token, newUserData) {
+  const CREATE_URL = process.env.VITE_FR_AM_CREATE_USER_URL;
+
+  if (!CREATE_URL) {
+    throw new Error('You must provide an CREATE_URL');
+  }
+  if (!access_token) {
+    throw new Error('No access token provided, exiting without a network call');
+  }
+
+  try {
+    const response = await fetch(CREATE_URL, {
+      method: 'POST',
+      headers: {
+        Authorization: `Bearer ${access_token}`,
+        'Content-Type': 'application/json',
+        Accept: '*/*',
+        Connection: 'keep-alive',
+      },
+      body: JSON.stringify(newUserData),
+    });
+    console.log('CREATE_USER_RESPONSE', response.statusText);
+    const resBody = await response.json();
+    return resBody;
+  } catch (err) {
+    console.log(err);
+  }
+}
+
+async function deleteUser(access_token, userId) {
+  const DELETE_URL = `${process.env.VITE_FR_AM_CREATE_DELETE_URL}${userId}`;
```

Comment:
Since this is a new `env` variable, Lets make sure we add it to the .env.example file and comment why its there

Also - because this variable isn't going to be needed by `vite` we probably do not need to prepend `vite` to it as that is specific to the bundler and its requirements loading variables

Reply:
i guess the second reply was meant for the below comment. As for the first one, i will do it. It was on my mind and forgot to di it.

Reply:
I don't controle the creating of the new demo users, it is done by the test itself. Although I did had a beforeAll() hook for the login tests where i created the new demo user, we decided to remove it. So having said that, for the tests that performs the registration i only use the afterAll() hook. I could enable _test retries_ if you want.
FYI: I run all of the tests and succeeded.
 
<img width="449" alt="image" src="https://github.com/ForgeRock/forgerock-web-login-framework/assets/156847686/74bf53d7-72d7-44cd-9e44-9790cc537b4d">